### ISSUE TITLE
TFLiteConverter.

### ISSUE THREAD

**Issue by dhirajgite** [tensorflow/tensorflow#39704]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):Windows 10
- TensorFlow installed from (source or binary): Pip
- TensorFlow version (or github SHA if from source): 2.1.0

**Command used to run the converter or code if you’re using the Python API**
If possible, please share a link to Colab/Jupyter/any notebook.

```
converter = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model = converter.convert()
```

**The output from the converter invocation**

```
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: CAST, CONCATENATION, CONV_2D, FULLY_CONNECTED, GATHER, RESHAPE, REVERSE_V2, SOFTMAX, TRANSPOSE. Here is a list of operators for which you will need custom implementations: TensorListFromTensor, TensorListReserve, TensorListStack, While.
```

**Also, please include a link to the saved model or GraphDef**

```
# Put link here or attach to the issue.
```

**Failure details**
If the conversion is successful, but the generated model is wrong,
state what is wrong:
- Producing wrong results and/or decrease in accuracy
- Producing correct results, but the model is slower than expected (model generated from old converter)

**RNN conversion support**
If converting TF RNN to TFLite fused RNN ops, please prefix [RNN] in the title.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by amahendrakar** [comment]
@dhirajgite,
In order to expedite the trouble-shooting process, could you please provide the complete code to reproduce the issue reported here.

Also, please take a look at [this comment](https://github.com/tensorflow/tensorflow/issues/39464#issuecomment-630018299) from a similar issue and let us know if it helps. Thanks!

**Comment by dhirajgite** [comment]
# Word Embedding
word_ip = Input(shape=(max_len,),name="Word_Embedding")
word_emb = Embedding(input_dim=n_words, output_dim=embedding_size, input_length=max_len)(word_ip)

# Next Word Embedding
next_word_ip = Input(shape=(max_len,),name="Next_Word_Embedding")
next_word_emb = Embedding(input_dim=n_words, output_dim=embedding_size, input_length=max_len)(next_word_ip)

# Prev Word Embedding
prev_word_ip = Input(shape=(max_len,),name="Prev_Word_Embedding")
prev_word_emb = Embedding(input_dim=n_words, output_dim=embedding_size, input_length=max_len)(prev_word_ip)

# POS Embedding
pos_ip = Input(shape=(max_len,),name="POS_Embedding")
pos_emb = Embedding(input_dim=n_pos, output_dim=embedding_size, input_length=max_len)(pos_ip)

#Next Pos Embedding
next_pos_ip = Input(shape=(max_len,),name="Next_POS_Embedding")
next_pos_emb = Embedding(input_dim=n_pos, output_dim=embedding_size, input_length=max_len)(next_pos_ip)

# Prev Pos Embedding
prev_pos_ip = Input(shape=(max_len,),name="Prev_POS_Embedding")
prev_pos_emb = Embedding(input_dim=n_pos, output_dim=embedding_size, input_length=max_len)(prev_pos_ip)

# Char Embedding
char_in = Input(shape=(max_len, max_len_char,),name = "Char_Embedding")
char_emb = TimeDistributed(Embedding(input_dim=n_chars+2, output_dim=50,
                           input_length=max_len_char))(char_in)
char_cnn = TimeDistributed(Conv1D(filters=53,kernel_size=3))(char_emb)
# char_cnn = TimeDistributed(LeakyReLU(alpha=0.3))(char_cnn)
# char_cnn = TimeDistributed(Dropout(rate=0.68))
# char_cnn = TimeDistributed(MaxPooling1D(pool_size=3))(char_cnn)
char_cnn = TimeDistributed(Flatten())(char_cnn)

# Char Feature Embedding
char_ftr_in = Input(shape=(max_len, max_len_char,),name="Char_ftr_embedding")
char_ftr_emb = TimeDistributed(Embedding(input_dim=n_char_ftr, output_dim=50,
                           input_length=max_len_char))(char_ftr_in)
char_ftr_cnn = TimeDistributed(Conv1D(filters=53,kernel_size=3))(char_ftr_emb)
char_ftr_cnn = TimeDistributed(Flatten())(char_ftr_cnn)
# Word Features                               
word_ftr_ip = Input(shape=(max_len,),name="Word_ftr_Embedding")
word_ftr_emb = Embedding(input_dim=n_word_ftr, output_dim=embedding_size, input_length=max_len)(word_ftr_ip)

lstm_ip = concatenate([word_emb,next_word_emb,prev_word_emb, pos_emb,next_pos_emb, prev_pos_emb, char_cnn, char_ftr_cnn, word_ftr_emb])
inputs = [word_ip,next_word_ip,prev_word_ip,pos_ip,next_pos_ip,prev_pos_ip,char_in,char_ftr_in,word_ftr_ip]

# Main LSTM Layer

out = Bidirectional(LSTM(units=275, return_sequences=True))(lstm_ip)
out = TimeDistributed(Dense(n_tags, activation="softmax"))(out) 
model = Model(inputs, out)
model.summary()

**Comment by dhirajgite** [comment]
word_ip = Input(shape=(max_len,),name="Word_Embedding")
word_emb = Embedding(input_dim=n_words, output_dim=embedding_size, input_length=max_len)(word_ip)


next_word_ip = Input(shape=(max_len,),name="Next_Word_Embedding")
next_word_emb = Embedding(input_dim=n_words, output_dim=embedding_size, input_length=max_len)(next_word_ip)


prev_word_ip = Input(shape=(max_len,),name="Prev_Word_Embedding")
prev_word_emb = Embedding(input_dim=n_words, output_dim=embedding_size, input_length=max_len)(prev_word_ip)


pos_ip = Input(shape=(max_len,),name="POS_Embedding")
pos_emb = Embedding(input_dim=n_pos, output_dim=embedding_size, input_length=max_len)(pos_ip)


next_pos_ip = Input(shape=(max_len,),name="Next_POS_Embedding")
next_pos_emb = Embedding(input_dim=n_pos, output_dim=embedding_size, input_length=max_len)(next_pos_ip)

prev_pos_ip = Input(shape=(max_len,),name="Prev_POS_Embedding")
prev_pos_emb = Embedding(input_dim=n_pos, output_dim=embedding_size, input_length=max_len)(prev_pos_ip)


char_in = Input(shape=(max_len, max_len_char,),name = "Char_Embedding")
char_emb = TimeDistributed(Embedding(input_dim=n_chars+2, output_dim=50,
                           input_length=max_len_char))(char_in)
char_cnn = TimeDistributed(Conv1D(filters=53,kernel_size=3))(char_emb)
char_cnn = TimeDistributed(Flatten())(char_cnn)


char_ftr_in = Input(shape=(max_len, max_len_char,),name="Char_ftr_embedding")
char_ftr_emb = TimeDistributed(Embedding(input_dim=n_char_ftr, output_dim=50,
                           input_length=max_len_char))(char_ftr_in)
char_ftr_cnn = TimeDistributed(Conv1D(filters=53,kernel_size=3))(char_ftr_emb)
char_ftr_cnn = TimeDistributed(Flatten())(char_ftr_cnn)
                             
word_ftr_ip = Input(shape=(max_len,),name="Word_ftr_Embedding")
word_ftr_emb = Embedding(input_dim=n_word_ftr, output_dim=embedding_size, input_length=max_len)(word_ftr_ip)

lstm_ip = concatenate([word_emb,next_word_emb,prev_word_emb, pos_emb,next_pos_emb, prev_pos_emb, char_cnn, char_ftr_cnn, word_ftr_emb])
inputs = [word_ip,next_word_ip,prev_word_ip,pos_ip,next_pos_ip,prev_pos_ip,char_in,char_ftr_in,word_ftr_ip]



out = Bidirectional(LSTM(units=275, return_sequences=True))(lstm_ip)
out = TimeDistributed(Dense(n_tags, activation="softmax"))(out) 
model = Model(inputs, out)
model.summary()

**Comment by dhirajgite** [comment]
Tried suggestion from comment worked
